flink-end-to-end-tests/test-scripts/test_tpcds.sh

```diff
@@ -66,10 +66,10 @@ function run_test() {
         set_config_key "taskmanager.numberOfTaskSlots" "4"
     elif [ "${scheduler}" == "AdaptiveBatch" ]; then
         set_config_key "taskmanager.numberOfTaskSlots" "8"
-        set_config_key "jobmanager.adaptive-batch-scheduler.max-parallelism" "8"
-        set_config_key "jobmanager.adaptive-batch-scheduler.avg-data-volume-per-task" "6m"
-        set_config_key "jobmanager.adaptive-batch-scheduler.speculative.enabled" "true"
-        set_config_key "jobmanager.adaptive-batch-scheduler.speculative.block-slow-node-duration" "0s"
+        set_config_key "execution.batch.adaptive.auto-parallelism.max-parallelism" "8"
+        set_config_key "execution.batch.adaptive.auto-parallelism.avg-data-volume-per-task" "6m"
+        set_config_key "execution.batch.adaptive.speculative.enabled" "true"
+        set_config_key "execution.batch.adaptive.speculative.block-slow-node-duration" "0s"
```

Comment:
We should wait the [PR21838](https://github.com/apache/flink/pull/21838) to be merged, and change the option name here.

Reply:
updated!